src/resspect/fit_lightcurves.py

```diff
@@ -130,14 +122,14 @@ def fit_snpcc(
         File names prefix
      number_of_processors: int, default 1
         Number of cpu processes to use.
-     feature_extractor: str, default bazin
+     feature_extractor: str, default Bazin
         Function used for feature extraction.
     """
-    if feature_extractor == 'bazin':
+    if feature_extractor == 'Bazin':
```

Comment:
Is there a common interface to be factored out from feature extraction classes here?

Reply:
Yes, you are correct, I do think that this will eventually be factored out, but not in this PR. There's an issue that covers this work here: https://github.com/LSSTDESC/RESSPECT/issues/64